modules/platform/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/operations/services/TestFileManagerImport.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Mickaël Schoentgen <mschoentgen@nuxeo.com>
+ */
+
+package org.nuxeo.ecm.automation.core.operations.services;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.automation.AutomationService;
+import org.nuxeo.ecm.automation.OperationContext;
+import org.nuxeo.ecm.automation.OperationException;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.test.CoreFeature;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.TransactionalFeature;
+
+/**
+ * @since 11.2
+ */
+@RunWith(FeaturesRunner.class)
+@Features(CoreFeature.class)
+@Deploy("org.nuxeo.ecm.platform.types.api")
+@Deploy("org.nuxeo.ecm.platform.types.core")
+@Deploy("org.nuxeo.ecm.platform.filemanager.api")
+@Deploy("org.nuxeo.ecm.platform.filemanager.core")
+@Deploy("org.nuxeo.ecm.automation.core")
+@Deploy("org.nuxeo.ecm.automation.features")
+@Deploy("org.nuxeo.ecm.platform.webapp.types")
+public class TestFileManagerImport {
+
+    @Inject
+    protected CoreSession session;
+
+    @Inject
+    protected AutomationService service;
+
+    @Inject
+    public TransactionalFeature txFeature;
+
+    protected DocumentModel parent;
+
+    @Before
+    public void before() {
+        parent = session.createDocumentModel("/", "foo", "File");
+        parent = session.createDocument(parent);
+    }
+
+    @Test
+    public void testFileImport() throws OperationException {
+        try (OperationContext ctx = new OperationContext(session)) {
+            ctx.setInput(parent);
+            Map<String, Serializable> params = new HashMap<>();
+            params.put("title", "bar");
```

Comment:
```suggestion
            Map<String, Serializable> params = Map.of("title", "bar");
```

Same for other Maps.